tools/medaka/convert_VCF_info_fields.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+
+# Takes in VCF file annotated with medaka tools annotate and converts
+#
+# Usage statement:
+# python convert_VCF_info_fields.py in_vcf.vcf out_vcf.vcf
+
+# 10/21/2020 - Nathan P. Roach, natproach@gmail.com
+
+import sys
+from math import log10
+
+from scipy.stats import fisher_exact
+
+
+def pval_to_phredqual(pval):
+    return int(round(-10. * log10(pval)))
```

Comment:
```suggestion
    return round(-10 * log10(pval))
```

but is also used only once, so could be inlined.